test/jdk/javax/swing/JToggleButton/bug4128979.java

```diff
@@ -22,63 +22,67 @@
  */
 
 /* @test
-   @bug 4128979
-   @requires (os.family == "windows")
-   @summary Tests that background changes correctly in WinLF for JToggleButton when pressed
-   @key headful
-   @run applet/manual=yesno bug4128979.html
-*/
+ * @bug 4128979
+ * @requires (os.family == "windows")
+ * @modules java.desktop/sun.awt
+ * @library /java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame jtreg.SkippedException
+ * @summary Tests that background changes correctly in WinLF for JToggleButton when pressed
+ * @run main/manual bug4128979
+ */
+
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.ComponentOrientation;
+import java.awt.Container;
+import java.awt.FlowLayout;
+import java.awt.Graphics;
 
 import javax.swing.BorderFactory;
 import javax.swing.BoxLayout;
 import javax.swing.Icon;
-import javax.swing.JApplet;
 import javax.swing.JFrame;
 import javax.swing.JLabel;
-import javax.swing.JOptionPane;
 import javax.swing.JPanel;
 import javax.swing.JToggleButton;
 import javax.swing.JToolBar;
 import javax.swing.UIManager;
-import javax.swing.UnsupportedLookAndFeelException;
-import javax.swing.WindowConstants;
-import java.awt.BorderLayout;
-import java.awt.Color;
-import java.awt.Component;
-import java.awt.ComponentOrientation;
-import java.awt.Container;
-import java.awt.FlowLayout;
-import java.awt.Graphics;
 
-public class bug4128979 extends JApplet {
+import jtreg.SkippedException;
+import sun.awt.OSInfo;
 
-    public static void main(String[] args) {
-        JApplet applet = new bug4128979();
-        applet.init();
-        applet.start();
+public class bug4128979 {
+    private static JFrame frame;
```

Comment:
`frame `can be moved inside `createAndShowUI()`.

Reply:
Updated.